### PR TITLE
V1 tasks

### PR DESCRIPTION
**Pull request by maartenscholz**

### CODE REVIEW

**Comment by maartenscholz** [comment]
No, good catch

File: apiary.apib [hunk 1]
```diff
@@ -1649,6 +1649,110 @@ Delete a meeting from Teamleader
...
+### addTask.php [POST /addTask.php]
+
++ Request (application/json)
+
+    + Attributes
+        + description: Description (string, required) - Description of the task
+        + due_date: 1265412302 (number, required) - Due date of the task as Unix timestamp
+        + `user_id`: 1 (number) - Id of the user (is required if team_id is not used)
+        + `team_id`: 2 (number) - Id of the team (is required if user_id is not used)
+        + task_type_id: 3 (number, required) - Category of the task
```
Comment (luisabritotl) [inline apiary.apib]: PP: change task_category or change the description to Type of tasks

File: apiary.apib [hunk 2]
```diff
@@ -1649,6 +1649,110 @@ Delete a meeting from Teamleader
...
+        + duration: 120 (number, required) - Amount of mintues that should be spent on the task
```
Comment (luisabritotl) [inline apiary.apib]: minutes

File: apiary.apib [hunk 3]
```diff
@@ -1649,6 +1649,110 @@ Delete a meeting from Teamleader
...
+                + C
+                + D
+        + for: company (enum[string]) - Who this task should be billed to
+            + Members
+                + contact
+                + company
+                + project_milestone
+        + for_id: 4 (number) - The id of who this task should be billed to
+        + creator_user_id: 5 (number) - Id the user that created the task
+        + related_deal_id: 6 (number) - Id of the deal related to the task
```
Comment (luisabritotl) [inline apiary.apib]: is the word related necessary?

File: apiary.apib [hunk 4]
```diff
@@ -1649,6 +1649,110 @@ Delete a meeting from Teamleader
...
+        29
+
+### updateTask.php [POST /updateTask.php]
+
++ Request (application/json)
+
+    + Attributes
+        + task_id: 1 (number, required) - Id of the task
+        + description: Description (string) - Description of the task
+        + duration: 120 (number) - Amount of mintues that should be spent on the task
```
Comment (luisabritotl) [inline apiary.apib]: minutes

File: apiary.apib [hunk 5]
```diff
@@ -1649,6 +1649,110 @@ Delete a meeting from Teamleader
...
+
+### getTasks.php [POST /getTasks.php]
+
++ Request (application/json)
+
+    + Attributes
+        + amount: `25` (number, required) - The amount of tasks returned per request
+        + pageno: `0` (number, required) - The current page
+        + selected_customfields: `42,142` (string) - Comma-separated list of the Id's of the custom fields you wish to select
+        + user_id: `1` (number) - Rreturn only tasks for this user
```
Comment (luisabritotl) [inline apiary.apib]: Return

File: apiary.apib (hunk 1 above)
Comment (maartenscholz) [inline apiary.apib]: I just copied the description from the old docs but I can update if that's what we are doing

File: apiary.apib (hunk 3 above)
Comment (maartenscholz) [inline apiary.apib]: Yeah, `deal to the task` is not really correct english in this context

**Pull request merged**